libcontainer/configs/validate/validator.go

```diff
@@ -7,7 +7,7 @@ import (
 	"strings"
 
 	"github.com/opencontainers/runc/libcontainer/configs"
-	"github.com/opencontainers/runc/libcontainer/selinux"
+	selinux "github.com/opencontainers/selinux/go-selinux"
```

Comment:
go-imports also makes it explicit, its fine both ways and is not an issue at all